app/templates/components/notification-dropdown.hbs

```diff
@@ -1,4 +1,4 @@
-<i class="mail outline icon {{unless this.unreadNotifications.length 'ui no right margin'}}"></i>
+{{!-- <i class="mail outline icon {{unless this.unreadNotifications.length 'ui no right margin'}}"></i>
```

Comment:
just change in nav-bar.hbs

Reply:
ok

Reply:
@SundaramDubey just changing in nav-bar does'nt resolves problem. i have to hide icon and app/notifications/unread
as well.